test/parallel/test-argv0-property.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.ok(process.hasOwnProperty('argv0'), 'argv0 property exists');
```

Comment:
`assert.strictEqual(process.hasOwnProperty('argv0'), true, 'argv0 should exists');`
The message is only used on failure, so it should have the negative of the assertion.